src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private String color;
+    private int radius;
+
+    public Circle(String color, int radius) {
+        this.color = color;
+        this.radius = radius;
+    }
+
+    @Override
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public void setColor(String color) {
+        this.color = color;
+    }
```

Comment:
you can use setters and getters from superclass by `super.getColor()`, so, don't override it